server/server.go

```diff
@@ -363,3 +371,20 @@ func (svr *Server) GetStoreStats(bt store.BackendType) (*store.Stats, error) {
 
 	return nil, fmt.Errorf("store not found")
 }
+
+type HandlerError struct {
+	Err  error
+	Meta *commitments.CommitmentMeta
```

Comment:
let's not use a pointer here. scares me in case this gets changed somewhere else.

Reply:
sounds great, I've removed the pointer